platform/pensando/sonic-platform-modules-dpu/dpu/utils/dpu_db_util.py

```diff
@@ -0,0 +1,602 @@
+#!/usr/bin/env python3
+# {C} Copyright 2023 AMD Systems Inc. All rights reserved
+#############################################################################
+# Pensando
+#############################################################################
+
+import sys
+import signal
+import subprocess
+import threading
+import time
+from datetime import datetime
+import json
+import docker
+import redis
+import syslog
+from sonic_py_common import daemon_base, logger, syslogger
+from swsscommon.swsscommon import SonicV2Connector
+from sonic_py_common.task_base import ProcessTaskBase
+import grpc
+from google.protobuf.empty_pb2 import Empty
+
+SYSLOG_IDENTIFIER = 'dpu-db-utild'
+logger_instance = syslogger.SysLogger(SYSLOG_IDENTIFIER)
+
+def log_info(msg, also_print_to_console=False):
+    logger_instance.log_info(msg, also_print_to_console)
+
+def log_err(msg, also_print_to_console=False):
+    logger_instance.log_error(msg, also_print_to_console)
+
+try:
+    from health_checker.manager import HealthCheckerManager
+    from sonic_py_common import daemon_base
+    import sonic_platform
+    from sonic_platform.chassis import Chassis
+    from sonic_platform.helper import APIHelper
+    from pathlib import Path
+    grpc_files_path = (str(Path(sonic_platform.__file__).parent.absolute()))
+    if grpc_files_path not in sys.path:
+        sys.path.append(grpc_files_path)
+    from sonic_platform import oper_pb2, oper_pb2_grpc, system_pb2, system_pb2_grpc
+except Exception as e:
+    log_err(f'failed to load modules due to {e}')
+
+DPU_HEALTH_INFO_TABLE_NAME = 'DPU_STATE'
+REBOOT_CAUSE_INFO_TABLE_NAME = 'REBOOT_CAUSE'
+VERSION_UPDATE_TABLE_NAME = 'PLATFORM|VERSION|CURRENT'
```

Comment:
@shanshri not aware of this table. seems propietary table

Reply:
@prgeor  same as the above.  This is stored into chassisStateDB once on DPU bootup.